src/routes/safe/components/Settings/SpendingLimit/NewLimitSteps.tsx

```diff
@@ -0,0 +1,79 @@
+import { Text } from '@gnosis.pm/safe-react-components'
+import React from 'react'
+import styled from 'styled-components'
+
+import Img from 'src/components/layout/Img'
+import AssetAmount from './assets/asset-amount.svg'
+import Beneficiary from './assets/beneficiary.svg'
+import Time from './assets/time.svg'
+
+const StepWrapper = styled.div`
```

Comment:
I suggest avoiding using both `StyledComponents` and `createStyles`, I know we are mixing it already on the code but would be better to add new components with just only `createStyles`

Reply:
but, why? @Agupane

Reply:
I don't like the idea of mixing different ways of handling with styles, there is a lot of that on the entire app, looks like we starting working in one way and then we changed and it's mixed now, maybe it will be a good idea to discuss this in another issue

Reply:
Agree on moving the discussion.

So far, as I can tell we're moving towards the styled-component approach. @alongoni can you confirm that?